docs/guide/src/pd/install.md

```diff
@@ -0,0 +1,32 @@
+# Installing pd
+
+Download prebuilt binaries from the [Penumbra releases page on Github](https://github.com/penumbra-zone/penumbra/releases).
+Make sure to use the most recent version available, as the version of `pd` must
+match the software currently running on the network.
+
+Make sure choose the correct platform for your machine. After downloading the `.tar.xz` file,
+extract it, and copy its contents to your `$PATH`. For example:
+
+```
+curl -O -L https://github.com/penumbra-zone/penumbra/releases/download/{{ #include ../penumbra_version.md }}/pd-x86_64-unknown-linux-gnu.tar.xz
```

Comment:
The `#include` helper is a dope trick and I'm glad we have it now: there's a single point of update to modify the version number across the whole guide.